codecov.yml

```
@@ -0,0 +1,11 @@
+coverage:
+  status:
+    project:
+      default: off
+      api:
+        target: 1% # Temopomary Testing Target
```

We should set a new temporary coverage target, because we set up a bunch of initial tests.
```suggestion
        target: 65% # Temopomary Testing Target
```